ocaml/xapi-cli-server/cli_frontend.ml

```diff
@@ -3597,6 +3597,33 @@ let rec cmdtable_data : (string * cmd_spec) list =
       ; flags= []
       }
     )
+  ; ( "tracing-create"
+    , {
+        reqd= ["name-label"]
+      ; optn=
+          [
+            "hosts"
+          ; "enabled"
+          ; "tags"
+          ; "endpoints"
+          ; "components"
+          ; "filters"
+          ; "processors"
+          ]
+      ; help= "Create a new TracerProvider."
```

Comment:
Ideally the machinery in the API would replicate the docstrings from the API, It might be useful to explain a bit what a traceprovider is

Reply:
Name changed to observer